site2/docs/security-oauth2.md

```diff
@@ -195,6 +195,22 @@ This example shows how to configure OAuth2 authentication in Node.js client.
 ```
 > Note: The support for OAuth2 authentication is only available in Node.js client 1.6.2 and later versions.
 
+## Broker configuration
+To configure brokers to authenticate clients, add the following parameters to broker.conf or standalone.conf:
+```properties
+# Configuration to enable authentication and authorization
+authenticationEnabled=true
+authorizationEnabled=true
```

Comment:
I think should be `authorizationEnabled=true` removed.

Reply:
You should remove any authorization config like `superUserRoles`, and update the description about authentication.

Reply:
@liangyepianzhou